packages/plugins/trpc/src/helpers.ts

```diff
@@ -21,14 +21,24 @@ export function generateProcedure(
         writer.write(`
         ${opType}: procedure.input(${typeName}).query(({ctx, input}) => checkRead(db(ctx).${lowerCaseFirst(
             modelName
-        )}.${prismaMethod}(input as any))),
+        )}.${prismaMethod}(input as any))) as ProcReturns<
+            "query",
+            Proc,
+            (typeof ${modelName}InputSchema)["${opType.replace('OrThrow', '')}"],
```

Comment:
I think the test failure was due to missing a `upperCaseFirst` call here. Prisma has some crazy casing behavior in generated types, and at the zod generation time, those are canonicalized to upper-case-first.

Reply:
```ts
aggregate: procedure
            .input(PubicFieldEntryInputSchema.aggregate)
            .query(({ ctx, input }) => checkRead(db(ctx).pubicFieldEntry.aggregate(input as any))) as ProcReturns<
            'query',
            Proc,
            (typeof PubicFieldEntryInputSchema)['aggregate'],
            ReturnType<PrismaClient['pubicFieldEntry']['aggregate']>
        >,
```

This is the generated code and it is working for me. Where exactly is `upperCaseFirst` missing?

Reply:
done ✔️

Reply:
weird...